http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/util/AKSInterpreter.scala

```diff
@@ -90,34 +96,52 @@ class AKSInterpreter[F[_]](config: AKSInterpreterConfig,
       )
       app = dbApp.app
       namespaceName = app.appResources.namespace.name
-
-      ksaName <- F.fromOption(
```

Comment:
I realized for Azure we were _never_ using Leo-created KSA, since aad-pod-identity doesn't require it. So I just removed it here, and from the setup chart (for Azure). In the new world WSM is creating the KSA.